docs/internals/howto-release-django.txt

```diff
@@ -129,18 +142,44 @@ any time leading up to the actual release:
 #. Double-check that the release notes index has a link to the notes
    for the new release; this will be in ``docs/releases/index.txt``.
 
-#. If this is a feature release, ensure translations from Transifex have been
-   integrated. This is typically done by a separate translation's manager
-   rather than the releaser, but here are the steps. Provided you have an
-   account on Transifex:
+#. If this is a feature release (i.e. the final release of a new version),
+   ensure translations from Transifex have been integrated. This is typically
+   done by a separate translation's manager rather than the releaser, but here
+   are the steps. This process is a bit lengthy and cumbersome, for example,
```

Comment:
I'd see much value in complaining in docs about out own process :upside_down_face: 
```suggestion
   are the steps.
```